server/server/search/tests/test_instant_search.py

```diff
@@ -120,4 +124,52 @@ def test_normalize_string():
     assert normalize_string('rupa') == 'rupa'
     assert normalize_string('Rūpā') == 'Rupa'
     assert normalize_string('samma') == 'samma'
-    assert normalize_string('sāmma') == 'samma'
\ No newline at end of file
+    assert normalize_string('sāmma') == 'samma'
+
+
+def instant_search(query, languages):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** No tests for unauthorized access.

It's important to ensure that the search API handles unauthorized access correctly, either by returning an appropriate error code or by not exposing sensitive information.

```suggestion
def test_unauthorized_access():
    response = client.post('/search/instant?query=test&language=en&limit=10&offset=0&matchpartial=false')
    assert response.status_code == 401 or response.status_code == 403
    assert 'error' in response.get_json()
```